documentation/plain_rules.md

```diff
@@ -156,7 +156,10 @@ This substitution can be used in the following fields:
 -   `id`, `type`, `name`, `value`, `ìsPattern` for `update` action
 
 Attribute value of `update` action and template of `post` action are expanded to numerical, boolean or JSON stringyfied
-values instead of string values when is possible.
+values instead of string values when is possible. For example, if we have `{"a": "${x}"}`:
+
+* If the value of attribute `x` is `42` then it will expand do `{"a": 42}` and not to `{"a": "42"}`
+* If the value of attribute `x` is `{"hello": "world"}` then it will expand to `{"a": "{/"hello/":/"world/"}"}` (expand to native JSON, i.e. `{"a": {"hello": "world"}}`, is not supported)
```

Comment:
Not sure if I have interpreted correctly what it's said at https://github.com/telefonicaid/perseo-fe/issues/362#issuecomment-536923473. Check this second bullet is 100% correct, pls.

Reply:
It should be \" instead of /"
i.e.: ``` {\"type\":\"Point\",\"coordinates\":[23,28]} ```

Reply:
Fixed by 70c1d60